dev-libs/libmelf/libmelf-0.4.0-r1.ebuild

```diff
@@ -11,30 +11,32 @@ SRC_URI="http://www.hick.org/code/skape/${PN}/${P}.tar.gz"
 LICENSE="Artistic"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
+IUSE="static-libs"
 
 DEPEND=""
 RDEPEND="${DEPEND}"
 
-src_prepare() {
-	# This patch was gained from the elfsign-0.2.2 release
-	epatch "${FILESDIR}"/${PN}-0.4.1-unfinal-release.patch
-	# Cleanup stuff
-	epatch "${FILESDIR}"/${PN}-0.4.0-r1-gcc-makefile-cleanup.patch
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.4.1-unfinal-release.patch
+	"${FILESDIR}"/${PN}-0.4.0-r1-gcc-makefile-cleanup.patch
+)
+
+src_configure() {
+	default
+	append-flags -fPIC
 }
 
 src_compile() {
-	append-flags -fPIC
-	emake CC="$(tc-getCC)" OPTFLAGS="${CFLAGS}" || die "emake failed"
+	emake CC="$(tc-getCC)" OPTFLAGS="${CFLAGS}" -Wl,soname,libmelf.so
 }
 
 src_install() {
 	into /usr
 	dobin tools/elfres
-	dolib.a libmelf.a
+	use static-libs && 	dolib.a libmelf.a
 	dolib.so libmelf.so
 	insinto /usr/include
 	doins melf.h stdelf.h
-	dodoc ChangeLog README
-	dohtml -r docs/html
+	HTML_DOCS=( docs/html )
```

Comment:
`HTML_DOCS=( docs/html/. )` (otherwise you have nested `html`)

Reply:
Done.  But doesn't seem to make any difference